content/meeting-notes/2017/pykc_coffee_20170513.md

```diff
@@ -0,0 +1,45 @@
+Title: PythonKC May 2017 Coffee & Code
+Date: 2017-05-13 00:00
+Category: Meeting Notes
+Tags: pythonkc, coffee & code
+Summary: Meeting notes for May 2017 PythonKC Coffee & Code
+
+```python
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from datetime import date
+today = date(2017, 5, 13)
+
+def greetings():
+    print("Welcome to PythonKC Coffee & Code.")
+    print("Today is {}".format(today.strftime("%A, %B %d, %Y")))
+    print("Thanks to the Johnson County Library system for hosting.")
+
+if __name__ == '__main__':
+    greetings()
+```
+
+## PythonKC Coffee & Code
+
+* [May 13th 2017](https://www.meetup.com/pythonkc/events/239247016/)
+* [JoCo Library, Cedar Roe](https://www.jocolibrary.org/locations/cedarroe)
+
+## Upcoming Events
+
+* [June TechTalk : KCWiT X Python KC Joint Meetup](https://www.eventbrite.com/e/june-techtalk-kcwit-x-python-kc-joint-meetup-tickets-32515143670?aff=website)
+* [Django Girls workshop](https://djangogirls.org/kansascity/)
+* [International Association for Social Science Information Services & Technology (IASSIST) Annual Conference](https://kupce.ku.edu/iassist-conference)
+* [DevOpsDays KC 2017 CFP](https://www.papercall.io/devopsdayskc)
+
+## Python News and Resources
+
+* [Py-backwards (Python to python compiler)](https://github.com/nvbn/py-backwards#py-backwards-)
+* [Write Fast Apps Using Async Python 3.6 and Redis](https://eng.paxos.com/write-fast-apps-using-async-python-3.6-and-redis)
+* [Asynchronous Programming in Python | Asyncio](http://djangostars.com/blog/asynchronous-programming-in-python-asyncio/)
+* [Queues in Python](https://dbader.org/blog/queues-in-python#)
+* [Generating Fake Data for Python Unit Tests with Faker](https://semaphoreci.com/community/tutorials/generating-fake-data-for-python-unit-tests-with-faker)
+* [Same Stats, Different Graphs](https://www.autodeskresearch.com/publications/samestats)
+* [ Beginner’s Guide to Neural Networks in Python](https://www.springboard.com/blog/beginners-guide-neural-network-in-python-scikit-learn-0-18/)
+* [osh (Object Shell)](https://github.com/geophile/osh)
+* [Statistical Data Analysis in Python (SciPy 2013)](https://github.com/fonnesbeck/statistical-analysis-python-tutorial)
```

Comment:
Can we add a section for links related to the command line stuff? Something like:

## Command Line Workshop Resources

* [Udacity Linux Command Line Basics](https://www.udacity.com/course/linux-command-line-basics--ud595) - A free course teaching the Linux command line.
* [Argparse Tutorial](https://docs.python.org/3.6/howto/argparse.html) - How to use Python's built-in `argparser` module to build command line applications.
* [Cloud9 IDE](https://c9.io/) - An easy, free way to get to a Linux command line if you don't have a Linux machine/VPS or feel like fighting with Virtual Machines.